docs/deploy-development-rp.md

```diff
@@ -176,6 +176,14 @@
   curl -X POST -k "https://localhost:8443/admin/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER/upgrade"
   ```
 
+* Get container logs from an OpenShift pod in a cluster
+  ```bash
+  NAMESPACE=<namespace-name>
+  POD=<pod-name>
+  CONTAINER=<container-name>
+  curl -X GET -k "https://localhost:8443/admin/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER/kubernetespodlogs?name=$POD&namespace=$NAMESPACE&container=$CONTAINER"
```

Comment:
Yes I think `podname` would be better.
And Yes `-X GET` is not necessary .. but since this is only GET method api .. I think keeping `-X GET` would give an idea about the API being only GET.